spec/generator/project_spec.rb

```diff
@@ -21,6 +21,28 @@ module Generator
         expect(@generator.project).not_to be_nil
       end
 
+      it 'should set class prefix' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.